packages/lib/services/interop/InteropService_Importer_Md.test.ts

```diff
@@ -1,21 +1,34 @@
 import InteropService_Importer_Md from '../../services/interop/InteropService_Importer_Md';
 import Note from '../../models/Note';
-import { setupDatabaseAndSynchronizer, supportDir, switchClient } from '../../testing/test-utils';
+import Folder from '../../models/Folder';
+import * as fs from 'fs-extra';
+import { createTempDir, setupDatabaseAndSynchronizer, supportDir, switchClient } from '../../testing/test-utils';
 import { MarkupToHtml } from '@joplin/renderer';
 
 
-describe('InteropService_Importer_Md: importLocalImages', function() {
+describe('InteropService_Importer_Md', function() {
+	let rootDir: string;
 	async function importNote(path: string) {
 		const importer = new InteropService_Importer_Md();
 		importer.setMetadata({ fileExtensions: ['md', 'html'] });
 		return await importer.importFile(path, 'notebook');
 	}
-
+	async function importNoteDirectory(path: string) {
+		const importer = new InteropService_Importer_Md();
+		importer.setMetadata({ fileExtensions: ['md', 'html'] });
+		return await importer.importDirectory(path, 'notebook');
+	}
+	beforeAll(async () => {
```

Comment:
Should be done in beforeEach / afterEach, otherwise you can't know if your tests are valid since they may use data from previous tests.

Reply:
Thanks for your review, I'm just creating a rootDir for testing here, all the tests runs in their sperate subdir. Therefore it should not be a issue. Or maybe it is better to create a temp folder for every test instead of one rootDir?